SDK v3 Examples/ios-sdk-app/ios-sdk-app/ViewController.swift

```diff
@@ -78,7 +79,11 @@ class ViewController: UIViewController, UIPickerViewDelegate, UIPickerViewDataSo
            let _selectedPolygon = selectedPolygon {
             //Get directions to selected polygon from users nearest node
             mapView?.getDirections(to: _selectedPolygon, from: _nearestNode, accessible: true) { directions in
-                self.mapView?.drawJourney(directions: directions, options: MPIOptions.Journey(connectionTemplateString: "<div style=\"font-size: 13px;display: flex; align-items: center; justify-content: center;\"><div style=\"margin: 10px;\">{{capitalize type}} {{#if isEntering}}to{{else}}from{{/if}} {{toMapName}}</div><div style=\"width: 40px; height: 40px; border-radius: 50%;background: green;display: flex;align-items: center;margin: 5px;margin-left: 0px;justify-content: center;\"><svg height=\"16\" viewBox=\"0 0 36 36\" width=\"16\"><g fill=\"white\">{{{icon}}}</g></svg></div></div>", pathOptions: MPIOptions.Path(color: "#cdcdcd", pulseColor: "#000000", displayArrowsOnPath: true)))
+                //remover custom markers before calling drawJourney
+                if let markerId = self.presentMarkerId {
+                    self.mapView?.removeMarker(id: markerId)
+                }
+                self.mapView?.drawJourney(directions: directions, options: MPIOptions.Journey(connectionTemplateString: "<div style=\"font-size: 13px;display: flex; align-items: center; justify-content: center;\"><div style=\"margin: 10px;\">{{capitalize type}} {{#if isEntering}}to{{else}}from{{/if}} {{toMapName}}</div><div style=\"width: 40px; height: 40px; border-radius: 50%;background: green;display: flex;align-items: center;margin: 5px;margin-left: 0px;justify-content: center;\"><svg height=\"16\" viewBox=\"0 0 36 36\" width=\"16\"><g fill=\"white\">{{{icon}}}</g></svg></div></div>", destinationMarkerTemplateString: nil, departureMarkerTemplateString: "", pathOptions: MPIOptions.Path(color: "#CDCDCD", pulseColor: "#000000", displayArrowsOnPath: true), polygonHighlightColor: "orange"))
```

Comment:
Can you put the parameters on a separate line since this one is super long

Reply:
I just changed them